src/legacy/server/config/schema.js

```diff
@@ -247,6 +247,9 @@ export default () =>
         applicationTitle: Joi.any().default(''),
         useExpandedHeader: Joi.boolean().default(true),
       }),
+      survey: Joi.object({
+        url: Joi.any().default('/'),
```

Comment:
why not: `https://amazonmr.au1.qualtrics.com/jfe/form/SV_1BxJNrtCo4LbweW`

Same as the default value above.

Reply:
I dont think we should add the amazon survey link in the code itself